content/spin/javascript-components.md

```diff
@@ -0,0 +1,279 @@
+title = "Building Spin Components in JavaScript"
+template = "spin_main"
+date = "2022-03-14T00:22:56Z"
+enable_shortcodes = true
+---
+
+With Javascript being a very popular language, Spin provides support for building components with it using the experimental SDK. The development of the Javascript SDK is continually being worked on to improve user experience and add features. 
```

Comment:
I think the "official" spelling is "JavaScript" (with a capital "S"), so we should adhere to that.